features/image-transformations/README.md

```diff
@@ -97,3 +97,19 @@ You can overlay text on an image and create dynamic banners. You can also contro
 {% endcontent-ref %}
 
 ![https://ik.imagekit.io/demo/tr:ot-overlay%20made%20easy,ots-45,otc-00FFFF/medium\_cafe\_B1iTdD0C.jpg](../../.gitbook/assets/text-overlay.jpg)
+
+### 7. SVG to raster format conversion
+
+ImageKit.io allows you to convert SVG images to raster formats by using the [format transformation parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
+
+To perform this conversion, you need to specify the desired raster format as `f-<raster_format>` in the image URL. For example: `f-png`, `f-jpg`, etc.
+
+{% hint style="info" %}
+If you enable automatic format conversion or use `f-auto`, the SVG images will not be converted to raster formats and will be delivered as SVG files only.
+{% endhint %}
+
+If you enable [automatic format conversion](../image-optimization/automatic-image-format-conversion.md) or use `f-auto` in the image URL, SVG images will not be converted to raster formats and will be delivered as SVG files only. Specifying `f-<raster_format>` in the URL is the only way to convert SVGs to raster formats.
```

Comment:
This line is the same as the `info` box right above.

Reply:
removed the infobox